### PR TITLE
OPSRC-496/Add bitbag coding standard and ecs fix

### PR DESCRIPTION
**Pull request by fabulousPuppet**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -45,18 +46,22 @@
...
+        "polishsymfonycommunity/symfony-mocker-container": "^1.0",
         "symfony/browser-kit": "^4.4 || ^5.2",
         "symfony/debug-bundle": "^4.4 || ^5.2",
         "symfony/dotenv": "^4.4 || ^5.2",
         "symfony/intl": "^4.4 || ^5.2",
         "symfony/web-profiler-bundle": "^4.4 || ^5.2",
         "symplify/console-color-diff": "^8.0 || ^9.3",
-        "vimeo/psalm": "4.6.4",
-        "polishsymfonycommunity/symfony-mocker-container": "^1.0"
+        "vimeo/psalm": "4.24"
```
Comment (senghe) [inline composer.json:56]: Why this change?

File: composer.json [hunk 2]
```diff
@@ -45,18 +46,22 @@
...
     },
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
+        "allow-plugins": {
+            "dealerdirect/phpcodesniffer-composer-installer": false,
+            "symfony/thanks": false,
+            "phpstan/extension-installer": false
+        }
```
Comment (senghe) [inline composer.json:64]: Why this changes?

File: composer.json (hunk 1 above)
Comment (fabulousPuppet) [reply composer.json:56]: "vimeo/psalm": "4.6.4" was too low version for "bitbag/coding-standard" so I updated it to the newest.

File: composer.json (hunk 2 above)
Comment (fabulousPuppet) [reply composer.json]: Oh, I missed it, ok I removed these changes.

**Review by senghe — APPROVED** [review]
